vitrina/structure/services.py

```diff
@@ -940,6 +944,18 @@ def _parse_access(value: str):
     return access
 
 
+def _parse_visibility(value: str):
+    if value:
+        return Visibility.objects.filter(title=value).first()
+    return Visibility.objects.filter(is_default=True).first()
+
+
+def _parse_status(value: str):
+    if value:
+        return Status.objects.filter(title=value).first()
+    return Status.objects.filter(is_default=True).first()
```

Comment:
- **Optional:** Seems to me like you are more `getting` than `parsing` here. Since parsing, at least to me, relates to the string type. I would suggest:
```suggestion
def _get_visibility(value: str) -> Visibility:
    if value:
        return Visibility.objects.filter(title=value).first()
    return Visibility.objects.filter(is_default=True).first()


def _get_status(value: str) -> Status:
    if value:
        return Status.objects.filter(title=value).first()
    return Status.objects.filter(is_default=True).first()
```

- Also, don't forget typehints for the return as well